process/economics/economicsData.go

```diff
@@ -19,14 +19,16 @@ type EconomicsData struct {
 	minGasLimit         uint64
```

Comment:
This file has a warning that the `EconomicsData` struct starts with the package name